docs/user-guide/docs-source/setup.md

```diff
@@ -0,0 +1,123 @@
+# Setup
+
+
+
+## Initial Setup (One-Time)
+
+Complete the following setup before using Ambit for the first time.
+
+>  ⏱ Initial setup takes about 20 minutes.
+
+> 🛑 These instructions assume you already have Unreal Engine 4.27 installed. If you don't, you can [download Unreal Engine for free](https://www.unrealengine.com/en-US/download). Choose the Unreal Engine license that is appropriate for your team. Ambit will work with any Unreal Engine license.
+
+### 1. Download the Ambit plugin for UE4
+
+Download the latest pre-built release of the Ambit UE4 plugin from the GitHub [Releases section](https://github.com/aws-samples/aws-ambit-scenario-designer-ue4/releases).
+
+Unzip the file you downloaded. You may unzip it to any location that's convenient for you. It will unzip as a folder named "AwsAmbit".
+
+We'll return to the Ambit plugin in later steps.
+
+### 2. Install the AWS Command Line Interface
+
+You will be configuring your computer to communicate with AWS services. The easiest way to do this is by installing the AWS Command Line Interface (CLI).
```

Comment:
"AWS Command Line Interface (CLI)" => "AWS Command Line Interface (AWS CLI)" - https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html
If we wish to keep just "CLI," we should change line #25 to be consistent.